request_information_test.go

```diff
@@ -141,18 +141,6 @@ func TestItDoesNotSetEmptySelectQueryParameters(t *testing.T) {
 	assert.Equal(t, "http://localhost/me", resultUri.String())
 }
 
-func TestItDoesNotSetEmptySearchQueryParameters(t *testing.T) {
```

Comment:
Any chance we could keep/modify the test and assert that the url string contains the query parameter with an empty value?

Reply:
I think I can modify it.

Reply:
I fix it in the latest commit.